lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -932,9 +932,12 @@ public function usesSequenceEmulatedIdentityColumns()
     public function getIdentitySequenceName($tableName, $columnName)
     {
         $table = new Identifier($tableName);
+        $column = new Identifier($columnName);
 
         // No usage of column name to preserve BC compatibility with <2.5
-        $identitySequenceName = $table->getName() . '_SEQ';
+        // ?????????
+
+        $identitySequenceName = $table->getName() . ($column->getName() !== '' ? '_' . $column->getName() : '') . '_SEQ';
```

Comment:
@Gemorroj you have to clarify about the actual error you encounter. Identity sequences created by Doctrine should be detected and working. What is the name of the sequence in your DB and what sequence name is DBAL generating? What are the steps to reproduce your error here?